nvflare/app_opt/xgboost/histogram_based_v2/sec/client_handler.py

```diff
@@ -91,6 +92,9 @@ def _process_before_broadcast(self, fl_ctx: FLContext):
             self.info(fl_ctx, "no clear gh pairs - ignore")
             return
 
+        if not ipcl_imported:
+            return self._abort("Required package ipcl-python is missing", fl_ctx)
+
```

Comment:
Do we want this check here?
Could there be plugins that does not need ipcl?
Should we check this flag elsewhere?

Reply:
when we use cuda plugin, will we enter this "_process_before_broadcast" ?

Reply:
Yes, this is always entered. Instead of checking ipcl_imported, you could check the existence of self.adder because the adder will be needed. This way, it will also work even if the adder is based on cuda.

Reply:
We need to check here because we don't know which plugin goes through this flow. 

Modified to check encryptor instead of ipcl is imported.